### PR TITLE
chore: more LNL ACs

### LINKED ISSUE #1275: LNL/ Snapshots ACs to be added
**Checkpoint LNL**
Settlement market going to LNL checkpoint  - shouldn’t come up
Define market with termination trigger - post LNL can then terminate
market_details match pre checkpoint. -stretch
Settled market going to LNL checkpoint  - shouldn’t come up

### PR DESCRIPTION
**Pull request by davidsiska-vega**
Closes #1275

### CODE REVIEW

File: protocol/0073-LIMN-limited_network_life.md [hunk 1]
```diff
@@ -334,3 +334,60 @@ Can deposit and withdraw funds to/from ERC20 asset loaded from checkpoint (<a na
...
+1. The LP bond account balance has been transferred to the party's general account for the asset. (<a name="0073-LIMN-033" href="#0073-LIMN-033">0073-LIMN-033</a>)
+
+
+## Test case 15: Market with trading terminated that settled is not restored, collateral moved correctly 
+1. Propose, enact, trade in the market, submit trading terminated and settlement data, observe final settlement cashflows for at least 2 parties. 
+1. System saves LNL checkpoint.
+1. Restart Vega, load LNL checkpoint. 
+1. The market is not restored (it doesn't exist in core i.e. it's not possible to submit orders or LP provisions to this market) (<a name="0073-LIMN-034" href="#0073-LIMN-034">0073-LIMN-034</a>) 
+1. If the market exists in the data node it is marked as settled with correct settlement data. (<a name="0073-LIMN-035" href="#0073-LIMN-035">0073-LIMN-035</a>)
+1. For parties that had margin balance position on the market this is now in their general account for the asset.  (<a name="0073-LIMN-036" href="#0073-LIMN-036">0073-LIMN-036</a>)
```
Comment (jgsbennett) [inline protocol/0073-LIMN-limited_network_life.md]: If the market is already settled, is it possible for parties to have margin? Same for insurance pool. (Or is this just saying to check that they don't magically reappear after checkpoint?)

File: protocol/0073-LIMN-limited_network_life.md [hunk 2]
```diff
@@ -334,3 +334,60 @@ Can deposit and withdraw funds to/from ERC20 asset loaded from checkpoint (<a na
...
+
+
+
+## Test case 14: Market with trading terminated is not restored, collateral moved correctly 
+1. Set LP fee distribution time step to non-zero value.
+1. Propose, enact, trade in the market, close out distressed party so that insurance pool balance > 0, submit trading terminated. 
+1. System saves LNL checkpoint at a time when undistributed LP fees for the market are > 0.
+1. Restart Vega, load LNL checkpoint. 
+1. The market is not restored (it doesn't exist in core i.e. it's not possible to submit orders or LP provisions to this market) (<a name="0073-LIMN-029" href="#0073-LIMN-029">0073-LIMN-029</a>) 
+1. If the market exists in the data node it is marked as settled with no settlement price info (<a name="0073-LIMN-030" href="#0073-LIMN-030">0073-LIMN-030</a>)
```
Comment (jgsbennett) [inline protocol/0073-LIMN-limited_network_life.md:346]: Duplicate AC code between this and the one below unfortunately.

File: protocol/0073-LIMN-limited_network_life.md (hunk 1 above)
Comment (davidsiska-vega) [reply protocol/0073-LIMN-limited_network_life.md]: You're absolutely right, once the market settled they should have no margin balances etc. But also as you say 

> This just saying to check that they don't magically reappear after checkpoint.

File: protocol/0073-LIMN-limited_network_life.md (hunk 2 above)
Comment (davidsiska-vega) [reply protocol/0073-LIMN-limited_network_life.md:346]: Oh bugger, that'll mess up the sequencing 🤣 .

**Review by jgsbennett — APPROVED** [review]

**Pull request merged**